docs/content/about-iota/iota-architecture/transaction-lifecycle.mdx

```diff
@@ -17,127 +17,107 @@ import ThemedImage from '@theme/ThemedImage';
     }}
 />
 
-1. **Create a Transaction**: A user [submits](#submission) a transaction and signs it with a private key. The transaction can affect objects owned by the user, as well as [shared objects](../../developer/iota-101/objects/object-ownership/shared.mdx).
+1.  **Create and Sign a Transaction**: A user creates a transaction using a wallet application. This transaction can affect objects owned by the user, as well as [shared objects](../../developer/iota-101/objects/object-ownership/shared.mdx). The wallet application then signs the transaction with the user's private key.
 
-2. **Submit to Validators**: The IOTA protocol sends the transaction to every validator. The validators validate the transaction. If valid, the validators sign it and return the signed transaction to the client.
+2.  **Submit to Validators**: The client application communicates with a full node, which then distributes the transaction to all validators following the IOTA protocol. The validators verify the transaction. If valid, the validators sign it and return the signed transaction to the full node.
 
-3. **Form a Transaction Certificate**: After collecting responses from a supermajority of validators, the client can form a transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
+3.  **Form an Aggregate Certificate**: After collecting responses from a supermajority of validators, the full node forms an aggregate transaction _certificate_. Unlike consensus-based blockchains, IOTA validators are not burdened by needing to propagate signatures or aggregate _certificates_; this is the responsibility of the client or gateway.
 
-4. **Send the Certificate**: The client sends the assembled _certificate_ back to all validators. The validators check its validity and acknowledge receipt. If the transaction involves only owned objects, IOTA can process and execute it immediately without waiting for consensus (**fast path consensus**). All _certificates_ are forwarded to the IOTA DAG-based consensus protocol.
+4.  **Send the Certificate**: The client sends the assembled aggregate _certificate_ back to all validators. The validators verify its validity and acknowledge receipt. If the transaction involves only owned objects, IOTA can process and execute it immediately without waiting for consensus (**fast path consensus**). All _certificates_ are forwarded to the IOTA DAG-based consensus protocol.
```

Comment:
I'm still a little confused. SUI always mention the client here. But above we changed that to full node. So I guess the full node still handles the certificate?

assembled aggregate seems to be a duplication of meaning btw. So again. I would just call it the assembled certificate